skins/laika/.env

```diff
@@ -0,0 +1,4 @@
+VUE_APP_LOGGER=console
+VUE_APP_ERRBIT_HOST=
+VUE_APP_ERRBIT_PROJECT_ID=
+VUE_APP_ERRBIT_PROJECT_KEY=
```

Comment:
If we're using env files, let's rely on Vue CLI's support of env files for different environments and create `env.development` and `env.production` files instead. The production environment should contain the errbit logger and access data.